nodejs/week1/src/backend/dataAccess.js

```diff
@@ -0,0 +1,11 @@
+const path = require("path");
+const fs = require("fs");
+
+exports.getDataFromJson = function (jsonPath) {
+  const filePath = path.join(__dirname, jsonPath);
+  let rawData = fs.readFileSync(filePath);
+  let jsonFile = JSON.parse(rawData);
+  return jsonFile;
+};
```

Comment:
You don't need to use the fs module to get some data from a json file. You can use `require` directly.

Reply:
Could you please clarify which require should I use directly?